### DISCUSSION TITLE
无法通过github登陆网站

### DISCUSSION THREAD

**Discussion by izumi-sora** [Gamuxorg/bbs#329] [游戏或网站运行反馈]
### 以下是推荐模板
游戏名称：网站登陆

操作系统：*（uname -a）*debian

bug描述：*（具体说说发生了什么）*网站无法通过github登陆

终端运行输出 ：*（如果有的话）* cURL error 28: Connection timed out after 10001 milliseconds Get access_token failed.

自行尝试解决bug的方式和结果：*（如果有的话）*清理Cookies and Site Data无效

**Comment by hyh19962008** [comment]
github最近访问不稳定，多重试几次就可以了

**Comment by shouhuanxiaoji** [comment]
你需要看一下，直接登陆github会不会有问题？网站的登陆功能，是网站在本地给github发验证消息。因此如果github访问不稳定就会出问题，请再验证一下
